config/routes.rb

```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
-  get "/articles", to: "articles#index"
-  get "/articles/:id", to: "articles#show"
+  # ----- Wheel Configuration ----- #
+  # get "/wheel", to: "wheels#index"
+  get "/prizes", to: "prizes#index"
+  get "/wheel-configurations", to: "wheel_configurations#index"
```

Comment:
If you don't need the `/wheel` endpoint you should remove it entirely.
Also, about the `# ----- Wheel Configuration ----- #`, I think this seems a bit redundant as this should be enough to understand what's happening in this case. Take a look [here](https://guides.rubyonrails.org/routing.html#controller-namespaces-and-routing), maybe you could do something like this:

```
scope '/wheel' do
  get "/prizes", ...
  get "/configuration", ...
end
```

Looking at it you wouldn't need the comment above as everything would be under the `wheel` scope.